Mathlib/Combinatorics/SimpleGraph/Connectivity.lean

```diff
@@ -2612,6 +2655,15 @@ instance : Decidable G.Connected := by
   rw [connected_iff, ← Finset.univ_nonempty_iff]
   infer_instance
 
+open Finset in
+instance Path.instFintype {u v : V} : Fintype (G.Path u v) where
+  elems := (univ (α := { p : G.Walk u v // p.IsPath ∧ p.length < Fintype.card V })).map
+    ⟨fun p ↦ { val := p.val, property := p.prop.left },
+     fun _ _ h ↦ SetCoe.ext <| Subtype.mk.injEq .. ▸ h⟩
+  complete p := mem_map.mpr ⟨
+    ⟨p.val, ⟨p.prop, p.prop.length_lt⟩⟩,
+    ⟨@mem_univ _ (fintypeSubtypePathLengthLt ..) _, rfl⟩⟩
```

Comment:
Here it complained with just `mem_univ _`: `failed to synthesize Fintype ↑fun p ↦ p.IsPath ∧ p.length < Fintype.card V` - I guess it is fine like this?

Reply:
I think it's fixed if you do...